src/DetailsView/components/test-view-container.tsx

```diff
@@ -64,6 +67,7 @@ export interface TestViewContainerProps {
     needsReviewCardsViewData: CardsViewModel;
     switcherNavConfiguration: DetailsViewSwitcherNavConfiguration;
     scanIncompleteWarnings: ScanIncompleteWarningId[];
+    tabStopRequirementData: TabStopRequirementState;
```

Comment:
This is actually not needed here.  The `ReportExportDialogFactory` is actually being passed into the the `TestViewContainer` via a prop of the `switcherNavConfiguration` property. This is basically the exact scenario we chatted about earlier! :)